src/client/src/ui/StatusBar/styled.tsx

```diff
@@ -88,10 +84,4 @@ export const ServiceRoot = styled.div`
   background-color: ${({ index = 0, theme }) =>
     // it'd be nice if this were selected from the original palette …
     darken(index / 50, theme.backgroundColor)};
-
-  ${Icon} {
```

Comment:
Don't remove functionality — provide the solution.

Additionally it is on the docket to get support for component selectors in TS — the better approach here is to make note of that.

Reply:
I have not removed this functionality. We will get TS running through babel at a later point and we can go back to doing things this way. I have added the margins back in the status bar file.